### DISCUSSION TITLE
featured_image not shown on Main

### DISCUSSION THREAD

**Discussion by DevTown** [gohugo-ananke/ananke#737] [2 - Q&A / Support] [answered]
Hi, i just upgraded the Theme to the version 2.10.0 and now i have the problem that the config setting:
featured_image = "/images/top.png"
will not be used any more. When I use it in a Blogentry the image will be shown like before but on the mainpage there is no image.
I used to set the image in the config.toml like featured_image = "/images/top.png" but now i cant find a way to reset this image.
What I am missing ?

**Comment by davidsneighbour** [comment]
I don't think we changed anything in this regard in 2.10. There is a PR about the featured image ongoing though... If you can provide a link to your repo or the website we can look into it. It might be a path issue, if your GoHugo site is in a subfolder. Running `hugo` and then checking `public/images` do you find `top.png`? If not, is it anywhere else inside of `public`?

**Reply by DevTown** [reply]
Hi,
the Website is dev-town.de. The Repo is privat. 
The image is in the image folder. I use the same image in some Blogposts and there it will be shown.
The main page do not habe the header HTML after i deploy it:
```html
Blogpost - > <header class="cover bg-top" style="background-image: url('http://localhost:1313/images/top.png');">
Mainpage ->  <header>
```

**Comment by davidsneighbour** [comment]
What is the front matter content of the file `content/_index.md`? The layout files do not look into the configuration; they only take either `featured_image` from the page's front matter or any image (the first one found) that contains `cover` or `feature` in its name.

You may have made a layout change that looked into the configuration and overrode it with the update. If you change theme files, you should always put a copy of the theme file with your changes in your repo root under `layouts` in the same place. 

I think it's generally a good idea to have a globally defined featured image, so we might add this to the features as well.

**Reply by DevTown** [reply]
I do not have a "content/_index.md"  file. 
I am used to work with the featured_image in the config.toml that will show the image as the headerimage. 
In the early version i used this like the feature you mentioned, a globally defined image. 
When i do not put a "featured_image" i a blog post it got the image i set in the config. 

I Came from version: e3f7fbe39d7b24d43e80a3c95aa3d940731ac535 (commit)

**Reply by DevTown** [reply, marked as answer]
Addin a _index.md to the content folder only with the featured_image Tag fixed the problem. 
THX. 
But the Feature to set a default Image could be a good feature ;-)
